### DISCUSSION TITLE
@incollection

### DISCUSSION THREAD

**Discussion by sejkko** [retorquere/zotero-better-bibtex#2455] [General]
This has probably been asked before, but I could not find an answer here. I can't seem to be able to cite a chapter of an edited book, which would be an @incollection entry in BibTeX using Zotero and Better BibTeX. I was wondering whether there is a way around this limitation. Editing the file produced by the export is not an option, of course. Thanks!

**Comment by retorquere** [comment]
I'll need a debug log for that. Right-click the entry, and select Better BibTeX -> Debug log from the popup menu.
